recipes/tcpcat/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include <tcpcat/tcpcat.h>
+
+#include <iostream>
+#include <string>
+
+int main()
+{
+   auto session = tcpcat::TcpSession(nullptr, nullptr, 0);
```

Comment:
I've simplified the test package to just create a tcpsession which does not instantiate any socket, we try to stay clear from the network layer in the tests, and this seems to work ok :)

Reply:
understandable, thank you!